src/main/kotlin/blackjack/model/Point.kt

```diff
@@ -0,0 +1,28 @@
+package blackjack.model
+
+import blackjack.dto.Number
+
+class Point(
+    private val numbers: List<Number>
+) {
+
+    fun getPoint(): Int {
+        var pointSet = mutableSetOf(0)
+        numbers.forEach {
+            val newPoint = mutableSetOf<Int>()
+            pointSet.addPoint(it.point, newPoint)
+            it.otherPoint?.let { point ->
+                pointSet.addPoint(point, newPoint)
+            }
+            pointSet = newPoint
+        }
+
+        return pointSet.filter { it <= 21 }.maxOrNull() ?: pointSet.min()
```

Comment:
```suggestion
        return pointSet.filter { it <= 21 }
            .maxOrNull() ?: pointSet.min()
```
객체지향 생활 체조 9가지 원칙
4. 한 줄에 점을 하나만 찍는다.